R/simulation-batch.R

```diff
@@ -71,6 +71,25 @@ SimulationBatch <- R6::R6Class(
       batchRunValues <- SimulationBatchRunValues$new(parameterValues, initialValues)
       rClr::clrCall(self$ref, "AddSimulationBatchRunValues", batchRunValues$ref)
       return(batchRunValues$id)
+    },
+
+    #' @description
+    #' Print the object to the console
+    #' @param ... Additional arguments.
+    print = function(...) {
+      simulationBatchOptions <- rClr::clrGet(self$ref, "SimulationBatchOptions")
+      private$printClass()
+      writeLines(paste0("   ", capture.output(self$simulation)))
```

Comment:
What does `capture.output` do?

Reply:
It sends output to a character string.

Reply:
What does that mean ? The result of print is concatenated to a string? Wouldn't printing the name of the simulation be enough?

Reply:
Yes, it looks like this:
```
SimulationBatch: 
   Simulation: 
      Name: Simple 
      Source file: D:/github/OSPSuite-R/tests/data/simple.pkml 
   runValuesIds: 
   Parameters: Organism|Liver|Volume R1|k1 
   Molecules: Organism|Liver|A 
```
But we could only have the name printed, sure.

Reply:
yeah. Let's remove the file name.  We do not need this I would think. Plus if we ever add more info in the simulation print., this is going to pollute this print